internal/environment/environment.go

```diff
@@ -0,0 +1,131 @@
+	DeafultModifyDataObjectIMG  = "quay.io/kubevirt/tekton-task-modify-data-object:" + operands.TektonTasksVersion
+	DeafultCopyTemplateIMG      = "quay.io/kubevirt/tekton-task-copy-template:" + operands.TektonTasksVersion
+	DeafultCleanupVMIMG         = "quay.io/kubevirt/tekton-task-execute-in-vm:" + operands.TektonTasksVersion
+	GenerateSSHKeysIMG          = "quay.io/kubevirt/tekton-task-generate-ssh-keys:" + operands.TektonTasksVersion
+	DefaultVirtioIMG            = "quay.io/kubevirt/virtio-container-disk:v0.59.0"
+
+	defaultOperatorVersion = "devel"
+)
+
+// GetSSHKeysStatusImage returns generate-ssh-keys task image url
+func GetSSHKeysStatusImage() string {
+	return EnvOrDefault(GenerateSSHKeysImageKey, GenerateSSHKeysIMG)
+}
+
+// GetWaitForVMIStatusImage returns wait-for-vmi-status task image url
+func GetWaitForVMIStatusImage() string {
+	return EnvOrDefault(WaitForVMISTatusImageKey, DefaultWaitForVMIStatusIMG)
+}
+
+// GetModifyVMTemplateImage returns modify-vm-template task image url
+func GetModifyVMTemplateImage() string {
+	return EnvOrDefault(ModifyVMTemplateImageKey, DeafultModifyVMTemplateIMG)
+}
+
+// GetDiskVirtSysprepImage returns disk-virt-sysprep task image url
+func GetDiskVirtSysprepImage() string {
+	return EnvOrDefault(DiskVirtSysprepImageKey, DeafultDiskVirtSysprepIMG)
+}
+
+// GetDiskVirtCustomizeImage returns disk-virt-customize task image url
+func GetDiskVirtCustomizeImage() string {
+	return EnvOrDefault(DiskVirtCustomizeImageKey, DeafultDiskVirtCustomizeIMG)
+}
+
+// GetCreateVMImage returns create-vm-from-manifest task image url
+func GetCreateVMImage() string {
+	return EnvOrDefault(CreateVMImageKey, DeafultCreateVMIMG)
+}
+
+// GetModifyDataObjectImage returns modify-data-object task image url
+func GetModifyDataObjectImage() string {
+	return EnvOrDefault(ModifyDataObjectImageKey, DeafultModifyDataObjectIMG)
+}
+
+// GetCopyTemplatemage returns copy-template task image url
+func GetCopyTemplateImage() string {
+	return EnvOrDefault(CopyTemplateImageKey, DeafultCopyTemplateIMG)
+}
+
+// GetCleanupVMImage returns cleanup-vm task image url
+func GetCleanupVMImage() string {
+	return EnvOrDefault(CleanupVMImageKey, DeafultCleanupVMIMG)
+}
+
+// GetVirtioImage returns virtio image url
+func GetVirtioImage() string {
+	return EnvOrDefault(VirtioImageKey, DefaultVirtioIMG)
+}
+
+// GetModifyVMTemplateImage returns modify-vm-template task image url
+func GetOperatorNamespace() string {
+	return EnvOrDefault(OperatorNamespaceKey, "kubevirt")
+}
+
+func LookupAsDuration(varName string) (time.Duration, error) {
+	duration := time.Duration(0)
+	varValue, ok := os.LookupEnv(varName)
+	if !ok {
+		return duration, errors.Errorf("Failed to load %s from environment", varName)
+	}
+
+	duration, err := time.ParseDuration(varValue)
+	if err != nil {
+		return duration, errors.Wrapf(err, "Failed to convert %s value to time.Duration", varName)
+	}
+	return duration, nil
+}
+
+func GetOperatorVersion() string {
```

Comment:
This fn already exists in https://github.com/kubevirt/ssp-operator/blob/master/internal/common/environment.go